plugin/Field/SearchcriteriaFields.php

```diff
@@ -119,9 +119,9 @@ public function getFieldLabelsOfInputs(array $inputFormFields): array
 
 			if (FieldTypes::isRangeType($pField->getType()))
 			{
-				if (__String::getNew($name)->endsWith(self::RANGE_FROM)) {
```

Comment:
Does this do the same? If so, I would prefer the `__String::getNew($name)->endsWith(self::RANGE_UPTO)` variant.